beestation.dme

```diff
@@ -14,6 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
+#include "_maps\map_files\EchoStation\job_changes.dm"
```

Comment:
Not meant to directly be included in the DME; this is why you had duplicate definition issues.
```suggestion
```